.github/workflows/release.yaml

```
@@ -114,17 +117,18 @@ jobs:
         id: release_body
         run: echo "release_body=$(ve1/bin/print-release-body)" | tee -a $GITHUB_OUTPUT
 
-      - name: Create the release
-        id: create_release
-        uses: softprops/action-gh-release@v2
-        with:
-          tag_name: ${{ steps.get_tag.outputs.release_version }}
-          body: ${{ steps.release_body.outputs.release_body }}
-          files: |
-            ${{ steps.build_bin.outputs.tarball_path }}
-            ${{ steps.generate_sbom_filename.outputs.sbom_filename }}
+      - name: Create tag and release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TAG_NAME: ${{ steps.get_tag.outputs.release_version }}
+          RELEASE_NOTES: ${{ steps.release_body.outputs.release_body }}
+          TARBALL_PATH: ${{ steps.build_bin.outputs.tarball_path }}
+          SBOM_FILENAME: ${{ steps.generate_sbom_filename.outputs.sbom_filename }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run:
```

```suggestion
        run: |
```

Think we should add the `|` multiline indicator here.

I don't think it's strictly necessary because we only have one command in scope, but because it's written with escaped line breaks, I'd say let's go ahead and add it for consistency.